tests/_craft/config/db.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
 return [
-    'password'    => getenv('DB_PASSWORD'),
-    'user'        => getenv('DB_USER'),
-    'database'    => getenv('DB_DATABASE'),
-    'tablePrefix' => getenv('DB_TABLE_PREFIX'),
-    'driver'      => getenv('DB_DRIVER'),
-    'port'        => getenv('DB_PORT'),
+    'password'    => 'root',
+    'user'        => 'root',
+    'database'    => 'scout_testing',
```

Comment:
n00b ?ion: Could explicitly defining DB and user creds here and elsewhere cause issues? Or is it explicitly re-defined per project?

Reply:
This is the test suite, so it's _fine_. Not necessarily ideal, would be better for it to still pull from env vars.